app/assets/javascripts/admin/addon/templates/user-index.hbs

```diff
@@ -350,7 +350,7 @@
       </div>
       <div class="controls">
         <DButton
-          @href="/admin/api/keys"
+          @href={{get-url "/admin/api/keys"}}
```

Comment:
Doesn't DButton handle that automatically? (if it doesn't: 1. it probably should, or 2. there are more cases to fix 👀)

Reply:
Ah. Let me confirm,